script.js

```diff
@@ -186,3 +186,47 @@ form.addEventListener('submit', (event) => {
     form.submit();
   }
 });
+
+// Form Input Local storage
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return (
+      e instanceof DOMException
+      && (e.code === 22
+        || e.code === 1014
+        || e.name === 'QuotaExceededError'
+        || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+      && storage
+      && storage.length !== 0
+    );
+  }
+}
+
+function populateStorage() {
+  localStorage.setItem('name', document.getElementById('name').value);
+  localStorage.setItem('email', document.getElementById('email').value);
+  localStorage.setItem('message', document.getElementById('message').value);
+}
```

Comment:
- [ ] I noticed that you have separate entries for each of the form inputs (name, email, and message).
Please ensure that the form data for all inputs are stored in a single object, saved to a single local storage registry as per requirements 👍 

![image](https://user-images.githubusercontent.com/46839548/232008914-4c0ce12d-1653-4ab6-bb93-a8df01f36967.png)

### HINT: 👇 
``` js
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```
Then turn it into a string before saving to local storage :
```js
localStorage.setItem('contactFormInput', JSON.stringify(formData));
```